configs/rover_configs.yaml

```diff
@@ -64,6 +64,37 @@ process_discovery:
         # The Process Labels, used to aggregate similar process from service entity
         # Multiple labels split by ","
         labels: ${ROVER_PROCESS_DISCOVERY_REGEX_SCANNER_LABELS:}
+  kubernetes:
+    # Is active the kubernetes process detector
+    active: ${ROVER_PROCESS_DISCOVERY_KUBERNETES_ACTIVE:false}
+    # Current node name in kubernetes environment
+    node_name: ${ROVER_PROCESS_DISCOVERY_KUBERNETES_NODE_NAME:}
+    # include namespaces, multiple namespace split by ",", if empty means including all namespaces
+    namespaces: ${ROVER_PROCESS_DISCOVERY_KUBERNETES_NAMESPACES:}
+    # Activated process analyze features
+    activated: ${ROVER_PROCESS_DISCOVERY_KUBERNETES_ACTIVATED:mesh,k8s}
+    # Customized the cluster name
+    cluster_name: ${ROVER_PROCESS_DISCOVERY_KUBERNETES_CLUSTER_NAME:k8s}
+    mesh:
+      # The name of service when analyze the process from mesh environment
+      service_name: ${ROVER_PROCESS_DISCOVERY_KUBERNETES_MESH_SERVICE_NAME:{{.Pod.Namespace}}::{{.Pod.LabelValue "service.istio.io/canonical-name,app.kubernetes.io/name,app" }}}
+    extend:
+      - filters:
+        # Extend process filter
+        - ${ROVER_PROCESS_DISCOVERY_KUBERNETES_EXTEND_FILTER:}
```

Comment:
What is this?

Reply:
It could let users dynamic add the customized process builder from the environment. Users need to add `extend` in `activated` to activate it.

Reply:
Could you provide an example? Such as how to use this? `Extend filter` seems don't make sense from naming perspective.

Reply:
`Extend Filter` means customized process builder.
It first needs to add the `extend` in the `activated` config.
Then config the filter and entity builder to let the rover know how to filter and build process entity.
Should I add an example in the use cases?